### PR TITLE
Feature/notes app

### PR DESCRIPTION
**Pull request by goutamiks**
issue No: #68
Screenshot of the output:

Notes app

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -202,8 +202,6 @@
             <div class="project-name">Email Validator</div>
             <a href="email validator/index.html" class="demo-link">Demo</a>
         </div>
-        <div class="table">
```
Comment (dhairyagothi) [inline public/index.html:205]: Update INDEX.html also

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by dhairyagothi** [comment]
@goutamiks PULL the MAIN branch before working on it

**Comment by dhairyagothi** [comment]
Good @goutamiks drop your email id so that i can give u score

**Comment by dhairyagothi** [comment]
@goutamiks check care from next time that you PULL main branch before doing any changes

**Comment by goutamiks** [comment]
Thank you.............. email id: goutamikodase@gmail.com

**Pull request merged**